omnigibson/prims/cloth_prim.py

```diff
@@ -194,11 +190,7 @@ def contact_list(self):
             list of CsRawData: raw contact info for this cloth body
         """
         contacts = []
-
-        # Avoid circular import
-        from omnigibson.systems.system_base import get_system_from_element_name
-        cloth_system = get_system_from_element_name("Cloth")
-
+        cloth_system = get_system("cloth")
```

Comment:
is it maybe better to have the cloth object store an internal reference to the (monolithic) cloth system instead of having to query get_system() every time?

Reply:
done